src/lib/moment/to-type.js

```diff
@@ -29,6 +29,6 @@ export function toObject () {
 }
 
 export function toJSON () {
-    // JSON.stringify(new Date(NaN)) === 'null'
-    return this.isValid() ? this.toISOString() : 'null';
+    // JSON.stringify(new Date(NaN)) === null
```

Comment:
The comment should just show `new Date(NaN).toJSON() === null`.  It's the `JSON.stringify` part that is confusing the issue.